### PR TITLE
adding a new project

### PR DESCRIPTION
**Pull request by YashasG98**
I think this will be a nice project to work on for beginners!

Please let me know if there are any errors in my template or if it needs any corrections

### CODE REVIEW

**Comment by YashasG98** [comment]
Changed the label.

File: _data/projects/multiplayer-game-server.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+  - python
+  - html
+  - css
+  - flask
+  - mysql
+  - javascript
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/YashasG98/Multiplayer-Game-Server/issues
```
Comment (shiftkey) [inline _data/projects/multiplayer-game-server.yml]: I think you can point this to the `up-for-grabs` label on the project:

```suggestion
  link: https://github.com/YashasG98/Multiplayer-Game-Server/labels/up-for-grabs
```

**Review by shiftkey — COMMENTED** [review]
@YashasG98 thanks for the contribution! 

Just one suggestion to point the link to the labelled issues.

**Comment by YashasG98** [comment]
I've made the suggested change :)

**Review by shiftkey — APPROVED** [review]

**Pull request merged**